pkg/acsengine/testdata/location/kubernetes.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
If location is specified, you should only produce certs for a single location in the output.